packages/salesforcedx-vscode-core/package.json

```diff
@@ -364,9 +371,49 @@
           "command": "sfdx.force.source.delete.current.file",
           "when": "sfdx:project_opened && sfdx:default_username_has_no_change_tracking && resourceLangId != 'forcesourcemanifest' && editorIsOpen"
         },
+        {
+          "command": "sfdx.force.set.default.org",
+          "when": "sfdx:project_opened"
```

Comment:
Is my understanding correct? - "sfdx:project_opened" is for regular dx projects that are opened, and sfdx:internal_dev is to surface commands even for internal development?

Reply:
Yes, you are correct. sfdx:internal_dev should only enable the new lightning create commands called `sfdx.internal.lightning.` and use the new entry methods which have the new preconditions and gatherers.